actions/rsasecureidam_enable_token.py

```diff
@@ -0,0 +1,35 @@
+# File: rsasecureidam_enable_token.py
+#
+# Copyright (c) 2023 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific language governing permissions
+# and limitations under the License.
+
+import phantom.app as phantom
+
+from actions import BaseAction
+from rsasecureidam_consts import RSA_ENABLE_TOKEN_QUERY, RSA_HEADER_LINE, RSA_TOKEN_ENABLE_MESSAGE
+
+
+class EnableToken(BaseAction):
+
+    def execute(self):
+        self._connector.save_progress("In action handler for: {0}".format(self._connector.get_action_identifier()))
+
+        token = self._param.get("token_serial")
```

Comment:
since `token_serial` is a required param, maybe better to use `self._param["token_serial"]` instead